### ISSUE TITLE
Add ingestion during scoring option to create task modal

### ISSUE THREAD

**Issue by jimmykodes** [codalab/competitions-v2#387]

**Comment by Didayolo** [comment]
Indeed, we can see [here](https://github.com/codalab/codabench/wiki/Yaml-Structure) that there is a YAML option `ingestion_only_during_scoring`. If true, the ingestion program will be run in parallel with the scoring program, and can communicate w/ the scoring program via a shared directory.

I think this possibility should be tested, as we never used it.

Adding the option in the "Create task" model would be look like this:

<img width="930" alt="Capture d’écran 2023-09-06 à 15 33 44" src="https://github.com/codalab/codabench/assets/11784999/91b64ca1-ffcb-4c07-861b-f80267760535">

**Comment by Didayolo** [comment]
@ObadaS @ihsaan-ullah 

Maybe we should remove the `ingestion_only_during_scoring` option?

**Comment by ihsaan-ullah** [comment]
> [@ObadaS](https://github.com/ObadaS) [@ihsaan-ullah](https://github.com/ihsaan-ullah)
> 
> Maybe we should remove the `ingestion_only_during_scoring` option?

I cannot think of a usecase where we would want to run ingestion and scoring in parallel. We either have ingestion followed by scoring, or only scoring.

In my opinion we don't need this feature

**Comment by ihsaan-ullah** [comment]
@Didayolo what have we decided about this `ingestion_only_during_scoring`. I am in favour of removing it to simplify things. If needed we can discuss this in the meeting next time.

**Comment by Didayolo** [comment]
Hi @ihsaan-ullah,

Indeed, I never used the feature, but Obada and Anne-Catherine convinced me that we wanted to keep it.

We have a "ingestion during scoring" template bundle to brush up that we can use to make tests and understand this feature:
https://github.com/codalab/competition-examples/tree/master/codabench/wheat_seeds/ingestion_during_scoring_bundle

**Comment by ihsaan-ullah** [comment]
I don't understand completely how what we do by using `ingestion_only_during_scoring` cannot be done using our regular `First Ingestion -> Then Scoring` flow.

But we can discuss further in the meeting with Obada and ACL.
